fragments/filter/README.md

```diff
@@ -0,0 +1,786 @@
+    "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson",
+    
+    "http://www.opengis.net/spec/ogcapi_common-2/1.0/req/collections",
+
+    "https://api.stacspec.org/v1.0.0-beta.1/item-search#filter:filter",
+    "https://api.stacspec.org/v1.0.0-beta.1/item-search#filter:features-filter",
+    "https://api.stacspec.org/v1.0.0-beta.1/item-search#filter:simple-cql",
+    "https://api.stacspec.org/v1.0.0-beta.1/item-search#filter:cql-text",
+    "https://api.stacspec.org/v1.0.0-beta.1/item-search#filter:cql-json",
+
+    "http://stacspec.org/spec/api/1.0.0-beta.1/core",
+    "http://stacspec.org/spec/api/1.0.0-beta.1/req/stac-search",
+    "http://stacspec.org/spec/api/1.0.0-beta.1/req/stac-response"
+  ],
+  "links": [
+    {
+      "title": "Search",
+      "href": "https://example.org/search",
+      "rel": "search",
+      "type": "application/geo+json"
+    },
+    {
+      "title": "Queryables",
+      "href": "https://example.org/queryables",
+      "rel": "http://www.opengis.net/def/rel/ogc/1.0/queryables",
+      "type": "application/schema+json"
+    }
+  ],
+  "stac_extensions": [],
+  "stac_version": "1.0.0",
+}
+```
+
+A client can use the link with `"rel": "http://www.opengis.net/def/rel/ogc/1.0/queryables"` to retrieve the queryables.
+
+The Queryables endpoint (`/queryables`) returns something like the following:
+
+```json
+{
+  "$schema" : "https://json-schema.org/draft/2019-09/schema",
+  "$id" : "https://example.org/queryables",
+  "type" : "object",
+  "title" : "Queryables for Example STAC API",
+  "description" : "Queryable names for the example STAC API Item Search filter.",
+  "properties" : {
+    "id" : {
+      "description" : "ID",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/id"
+    },
+    "collection" : {
+      "description" : "Collection",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/collection"
+    },
+    "geometry" : {
+      "description" : "Geometry",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/geometry"
+    },
+    "datetime" : {
+      "description" : "Datetime",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/datetime.json#/properties/datetime"
+    },
+    "eo:cloud_cover" : {
+      "description" : "Cloud Cover",
+      "$ref": "https://stac-extensions.github.io/eo/v1.0.0/schema.json#/properties/eo:cloud_cover"
+    },
+    "gsd" : {
+      "description" : "Ground Sample Distance",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/instrument.json#/properties/gsd"
+    },
+    "assets_bands" : {
+      "description" : "Asset eo:bands common names",
+      "$ref": "https://stac-extensions.github.io/eo/v1.0.0/schema.json#/properties/eo:bands/common_name"    
+    }
+  }
+}
+```
+
+Alternately, the client could retrieve the queryables for a single collection with 
```

Comment:
This wasn't clear to me from the Queryables section; consider mentioning this above. E.g.  "Queryables are advertised via a JSON Schema document retrieved from the /queryables endpoint." -> "Queryables are advertised via a JSON Schema document retrieved from the /queryables endpoint, either from the root to retrieve for the queryables across all collections, or off of a collection route (e.g. /collections/collection1/queryables) for the queryables pertaining to a single collection"

Is it required to implement all of the root queryables endpoint + collection-specific endpoints?

Reply:
Thanks, I clarified.